pkgs/by-name/pl/plumber/package.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "plumber";
+  version = "2.5.2";
+
+  src = fetchFromGitHub {
+    owner = "streamdal";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-ftXLipJQjRdOSNO56rIRfAKKU0kHtAK85hgcT3nYOKA=";
+  };
+
+  vendorHash = null;
+
+  # connection tests create a config file in user home directory
+  preCheck = ''
+    export HOME="$(mktemp -d)"
+  '';
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/streamdal/plumber/options.VERSION=${version}"
```

Comment:
should this set the version somehow? it doesn't seem to appear when using the command line.
```console
$ ./result/bin/plumber --version
UNSET
```

Reply:
I took this from the upstream Makefile. It also does not work there and I'm not sure why, tried some alternatives, i.e, just options.VERSION but also didn't work, though seems to me both should work just fine. :(

Reply:
[go.mod](https://github.com/streamdal/plumber/blob/90ee8c815f6dec674dd51602332012d83827f4f5/go.mod#L1C1-L1C36) sets `module github.com/batchcorp/plumber` so need to use `-X github.com/batchcorp/plumber/options.VERSION=${version}`

Reply:
```suggestion
    "-X github.com/streamdal/plumber/options.VERSION=${version}"
    # remove once module in go.mod is renamed to github.com/batchcorp/streamdal
    "-X github.com/batchcorp/plumber/options.VERSION=${version}"

```